package.json

```diff
@@ -111,18 +110,18 @@
     "@types/postcss-safe-parser": "^5.0.1",
     "@types/prismjs": "^1.26.0",
     "@types/stylus": "^0.48.38",
-    "@typescript-eslint/eslint-plugin": "^5.45.0",
-    "@typescript-eslint/parser": "^5.45.0",
-    "@typescript-eslint/parser-v4": "npm:@typescript-eslint/parser@4",
```

Comment:
Do we need this deps for now? I got a type error due to this deps, so I removed this.